### ISSUE TITLE
Sesame model

### ISSUE THREAD

**Issue by andomariot** [APSIMInitiative/ApsimX#7759]
UniSQ Centre for Sustainable Agricultural Systems is leading a project in Understanding Sesame with the Broadacre crop Initiative program (BACI) and partnership with QDAF.
The project is in its second year now and have collated field data  from central Queensland region ( 2017-2022) and is continuing  to collect data for the coming season 2022 to 2024 in collaboration with CQU.
Sufficient data may come then available to develop a sesame model with APSIM-X .
The approach will use the mungbean plant architecture as  model template but will learn more from oilseed plant modelling for processes, yield formation and oil content.
@APSIMInitiative/other-crops 
@APSIMInitiative/legumes

**Comment by hut104** [comment]
Hi
I wonder if the soybean model may be more appropriate given its oil content.
I would also recommend using newly developed ways to link these plant templates to avoid duplication of efforts.  I'd suggest looking into this once model development commences.

**Comment by tefera21** [comment]
UNOFFICIAL

Hi,

I also agree with Neil that a template of the soybean model is more appropriate for the Sesame-APSIM model development/parameterisation.

**Comment by hol353** [comment]
@APSIMInitiative/other-crops: FYI

**Comment by BrianCollinss** [comment]
More information will be soon provided regarding the parameterisation and validation datasets.
Contact: Brian Collins ([brian.collins@unisq.edu.au](mailto:brian.collins@unisq.edu.au), @BrianCollinss).

**Comment by BrianCollinss** [comment]
@hol353 the development of the Sesame model is under way. Some field data has been used to test the model and more is being collected. The model performs reasonably well so far. May I please know if it can be added to the Prototypes?

**Comment by hol353** [comment]
Absolutely. Prototypes is the place to put models that are being constructed.
